### ISSUE TITLE
Resources Learned (Continuous)

### ISSUE THREAD

**Issue by ahy69195** [ahy69195/AlbertYou_TasksQRG#8]
Explanation of Cost and Loss function:
https://medium.com/@vinodhb95/what-is-loss-in-neural-nets-is-cost-function-and-loss-function-are-same-ef069a570e95

Explanation of Linear regression regarding cost functions, gradient descent:
https://towardsdatascience.com/introduction-to-machine-learning-algorithms-linear-regression-14c4e325882a

**Comment by ahy69195** [comment]
Gradient descent, learning rate, check accuracy of gradient descent (plotting function as optimization runs to find convergence and check how good the learning rate is), types of gradient descent (batch, stochastic, mini-batch) 
https://builtin.com/data-science/gradient-descent

An image kernel is a small matrix used to apply effects like the ones you might find in Photoshop or Gimp, such as blurring, sharpening, outlining or embossing. They're also used in machine learning for 'feature extraction', a technique for determining the most important portions of an image. In this context the process is referred to more generally as "convolution" (see: convolutional neural networks.)
https://setosa.io/ev/image-kernels/

Types of convolution filters:
https://www.l3harrisgeospatial.com/docs/convolutionmorphologyfilters.html

1. Convolution in Convolutional Neural Networks
2. Convolution in Computer Vision
3. Power of Learned Filters
4. Worked Example of Convolutional Layers
https://machinelearningmastery.com/convolutional-layers-for-deep-learning-neural-networks/
